control-plane/stor-port/src/types/v0/transport/nexus.rs

```diff
@@ -242,6 +242,29 @@ pub struct CreateNexus {
     pub config: Option<NexusNvmfConfig>,
 }
 
+/// A request to resize a Nexus.
+#[derive(Serialize, Deserialize, Default, Debug, Clone, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct ResizeNexus {
+    /// Id of the io-engine instance
```

Comment:
Pls change others as well to end with `.`

```suggestion
    /// Id of the io-engine instance.
```

Reply:
> Pls change others as well to end with `.`

Modified. However, rest of the structs don't have it in the comments.

Reply:
They're probably "older", before we started following  this practice